rerun_cpp/src/rerun/archetypes/cylinders3d_ext.cpp

```diff
@@ -0,0 +1,24 @@
+#if 0
+
+#include "cylinders3d.hpp"
+
+namespace rerun::archetypes {
+    // <CODEGEN_COPY_TO_HEADER>
+
+    /// Creates a new `Cylinders3D` with the given axis-aligned lengths and radii.
+    ///
+    /// For multiple cylinders, you should generally follow this with
+    /// `Cylinders3D::with_centers()` and one of the rotation methods, in order to move them
+    /// apart from each other.
+    //
+    // TODO(andreas): This should not take an std::vector.
+    static Cylinders3D from_lengths_and_radii(
+        const std::vector<float>& lengths, const std::vector<float>& radii
+    ) {
+        return Cylinders3D().with_lengths(std::move(lengths)).with_radii(std::move(radii));
```

Comment:
I have no idea why I never followed on these. Let's try this from the start here:

```suggestion
    static Cylinders3D from_lengths_and_radii(
        const Collection<rerun::components::Length>& lengths, const Collection<rerun::components::Radius>& radii
            ) {
        return Cylinders3D().with_lengths(lengths).with_radii(radii);
```